plugin-animation/src/test/java/com/mapbox/maps/plugin/animation/CameraAnimationsPluginImplTest.kt

```diff
@@ -972,6 +974,43 @@ class CameraAnimationsPluginImplTest {
     unmockkStatic(ShadowLogger::class)
   }
 
+  @Test
+  fun catchErrorOnSetCameraTest() {
+    executeSetCameraTest(Error("Invalid camera options"))
+  }
+
+  @Test
+  fun catchExceptionOnSetCameraTest() {
+    executeSetCameraTest(Exception("Invalid camera options"))
+  }
+
+  private fun executeSetCameraTest(error: Throwable) {
+    val lastCameraOptions = cameraState.toCameraOptions()
+    val targetCameraOptions = CameraOptions.Builder().center(
+      Point.fromLngLat(50.0, 50.0)
+    ).pitch(50.0)
+      .bearing(50.0)
+      .zoom(5.0).build()
+    cameraAnimationsPluginImpl.lastCameraOptions = lastCameraOptions
+    every { mapCameraManagerDelegate.setCamera(any<CameraOptions>()) } throws error
+
+    cameraAnimationsPluginImpl.performMapJump(targetCameraOptions)
+    // assert camera options did not change
+    assertNotEquals(targetCameraOptions, cameraAnimationsPluginImpl.lastCameraOptions)
+
+    // assert camera states
+    assertNotNull(cameraAnimationsPluginImpl.lastCameraOptions)
```

Comment:
Instead of assert each field I think we can just assert _lastCameraOption_ and _cameraAnimationsPluginImpl.lastCameraOptions_ since CameraOptions has all checks in **equals** method